basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,102 @@
 const helpers = require('./project-1');
 
 // start testing!
+
+it('should multiply a number by 10', () => {
+    expect(helpers.multiplyByTen(10)).toBe(100);
+    expect(helpers.multiplyByTen(100)).toBe(1000);
+});
+
+it('should subtract a number by 5', () => {
+    expect(helpers.subtractFive(6)).toBe(1);
+    expect(helpers.subtractFive(5)).toBe(0);
+});
+ it('should check if two strings have equal lengths', () => {
+    expect(helpers.areSameLength('abc', 'def')).toBe(true);
+    expect(helpers.areSameLength('abcd', 'def')).toBe(false);
+});
+ it('should check if two numbers are the same', () => {
+    expect(helpers.areEqual(8, 9)).toBe(false);
+    expect(helpers.areEqual('8', '8')).toBe(true);
+});
+ it('should check if a number is greater than 50', () => {
+    expect(helpers.greaterThanFifty(49)).toBe(false);
+    expect(helpers.greaterThanFifty(50)).toBe(false);
+    expect(helpers.greaterThanFifty(51)).toBe(true);
```

Comment:
Good way to test edge cases, make the numbers close to the point where the output will change.